### PR TITLE
add landing message. #1

### PR DESCRIPTION
**Pull request by ramatulamin**
added landing message. #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
   <head>
-    <title>This is the title of the webpage!</title>
+    <title>Welcome to the collaborative project on Quantum!</title>
   </head>
   <body>
     <p>This is an example paragraph. Anything in the <strong>body</strong> tag will appear on the page, just like this <strong>p</strong> tag and its contents.</p>
   </body>
+
+  <body> 
+	  <h1> Welcome to potionfy! enjoy your stay</h1>
```
Comment (bashirmohd) [inline index.html]: could you remove the "enjoy your stay" string and make the "welcome to potionfy" string red?

Comment (bashirmohd) [reply index.html]: could you remove the "enjoy your stay" string and make the "welcome to potionfy" string red?

**Review by bashirmohd — COMMENTED** [review]
could you remove the "enjoy your stay" string and make the "welcome to potionfy" string red?

File: index.html (hunk 1 above)
Comment (ramatulamin) [reply index.html]: sure can do

**Review by bashirmohd — CHANGES_REQUESTED** [review]
please, change the code as per my comment

**Review by bashirmohd — APPROVED** [review]
well done Rammy

**Pull request merged**